fw/t/unit/test.c

```diff
@@ -90,6 +90,7 @@ TEST_SUITE(cfg);
 TEST_SUITE(tfw_str);
 TEST_SUITE(mem_fast);
 TEST_SUITE(http2_parser_hpack);
+TEST_SUITE(http_cache_key);
```

Comment:
Let's rename this to have common suite for all cache functions. 
```suggestion
TEST_SUITE(http_cache);
```